KDIC/BlogTableCell.swift

```diff
@@ -0,0 +1,23 @@
+//
```

Comment:
Remove these leading copyright notice comments (in all files). This information is incorrect and already stated in our license file.